exonum/src/api/node/public/explorer.rs

```diff
@@ -171,6 +172,80 @@ impl AsRef<[u8]> for TransactionHex {
     }
 }
 
+/// Status of a call.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct CallStatusResponse {
+    /// Call status
+    pub status: ExecutionStatus,
+    /// Call execution proof
+    pub call_proof: MapProof<CallInBlock, ExecutionError>,
```

Comment:
This is bad. We have a `serde` transform for `ExecutionError` (in `runtime::error::execution_error` module), but it cannot be applied here because `ExecutionError` is not a field. Then again, there was some talk about returning proofs in Protobuf. @aleksuss, can you comment on this?

Reply:
Yeah. We have a plan to return proofs in protobuf format but in scope of other ticket.

Reply:
I think that *currently* mixing `json` and `protobuf` will cause only inconvenience.
IMO for the proof contents the explicit deserialization format derived from the `rust` structure is not a bad thing: it's easier to write a parser depending on the structure layout (if you'll want to check the proof).

Reply:
Let's discuss this topic today on a grooming.